SparkPost/2web2ui
# FE-1118 - Update button variants from secondary to primary on the profile page

[FE-1118](https://jira.int.messagesystems.com/browse/FE-1118)

### What Changed
- Updated button variants from `"secondary"` to `"primary"` on the profile page

### How To Test
- Go to `/account/profile` to see the new buttons - they should be blue for Hibana and orange for the OG theme.

### To Do
